### ISSUE TITLE
Spuri: “Le casette sono da evitare Chi può scelga l’autonoma sistemazione”

### ISSUE THREAD

**Issue by terremotocentroita** [emergenzeHack/terremotocentro_segnalazioni#2550]
<pre><yamldata>titolo: 'Spuri: “Le casette sono da evitare Chi può scelga l’autonoma sistemazione”'
descrizione: '“Sono state presentate 1.200 perizie giurate riguardanti schede Aedes con esito B, l’unica tipologia per ora ammissibile – in attesa delle norme sui danni gravi e gravissimi che arriveranno entro qualche settimana –, ma sono giunti all’ufficio speciale per la ricostruzione soltanto nove progetti riguardanti l’edilizia abitativa e cinque per le attività commerciali. Il termine per la presentazione scade il 31 luglio, invito i tecnici a fare, invito a vincere l’iniziale titubanza e buttarsi. Non essendoci molte pratiche per noi è un buon test, un buon banco di prova. Se ci sono dubbi vanno espressi agli uffici e discussi insieme”. Con queste parole Cesare Spuri, responsabile dell’ufficio speciale per la ricostruzione delle Marche, ha descritto la situazione attuale della ricostruzione leggera per le schede Aedes con esito B, i Share j s Tweet h +1 f Share k Email o 0 SHARES cosiddetti danni lievi, di fronte alla platea che ha gremito questa mattina il cinema Italia di San Severino. Spuri ha invitato coloro che sono già in possesso della scheda Aedes a presentare i progetti e coloro che hanno avuto la scheda Fast con esito inutilizzabile o non agibile ad attivarsi subito con i tecnici per fare la perizia giurata. A fare da padrona di casa, è stata il sindaco Rosa Piermattei che ha annunciato la fine dei sopralluoghi Fast in città, prima tra i Comuni del cratere, così come per prima San Severino ha avviato le demolizioni all’Uvaiolo e nelle altre zone rosse. Mario Borri, sfollato del rione Uvaiolo ha annunciato la costituzione di un comitato, per approfondire il tema della ricostruzione dell’abitato della zona ed ha invitato tutti coloro che ne hanno la volontà, ad aderire. “Per San Severino abbiamo ordinato in tutto 98 casette – ha detto il primo cittadino . E’ stato detto che saranno consegnate a settembre e la popolazione è in allarme, certamente tra le 700 ordinate dalla Regione Marche ci saranno anche le nostre. Abbiamo terminato i sopralluoghi, abbiamo lavorato giorno e notte per evadere le oltre seimila richieste. Adesso ci preoccupa il risultato delle 1.457 schede Fast che danno gli edifici non utilizzabili. Non sappiamo cosa dovranno fare i cittadini e se dovremo richiedere altre casette. Lancio un appello ai tecnici affinché facciano velocemente le perizie. Abbiamo individuato tre aree per le Sae, abbiamo già parlato con i proprietari, speriamo di iniziare presto i lavori”. Spuri ha invitato ad andare avanti con le perizie giurate per accertare il danno su edifici già in possesso della scheda Fast: “In questa fase le perizie sono urgentissime, invito coloro che hanno ricevuto la squadra dei tecnici Fast con esito di inutilizzabilità a sbrigarsi, siamo tutti sulla stessa barca, adesso è il momento dei proprietari – ha detto Spuri –. Sulle casette non possiamo fare ordinativi a vuoto per una spesa di diversi milioni di euro, è danno erariale. Oltre alle Sae, l’altra opzione è di acquistare appartamenti, è già pronta la bozza, a breve uscirà il bando regionale. Le casette sono il male da evitare, consumano il suolo, comportano opere di lottizzazione vera e propria, finito l’uso vanno smaltite. Se consideriamo il costo medio al metro quadrato di una casetta di media entità, è facile paragonare i costi degli appartamenti invenduti, che ci portiamo dietro da dieci anni per la crisi dell’edilizia. Dopo l’occupazione dei terremotati, questi appartamenti potranno essere destinati ad altri usi. Se si ha un posto dove andare con il contributo di autonoma sistemazione, è meglio per tutti. Se potremo offrire gli appartamenti acquistati con il bando, li daremo a tutti, tra gli appartamenti e le casette scegliete questi, la Protezione civile acquisterà immobili sicuri, le casette sono l’ultima spiaggia”. Rispondendo alle domande dei presenti, Spuri ha spiegato che per le opere di urbanizzazione per le Sae (casette in legno) il termine previsto è di massimo venti giorni, si prevede un appalto senza interruzione e più turni di lavoro. All’ufficio speciale della ricostruzione sono arrivati una trentina di tecnici provenienti da Invitalia e Fintecna. “Abbiamo già ordinato settecento casette, la macchina continua a camminare anche se in modo sotterraneo – ha detto Spuri – è stata fatta la scelta di trasformare le schede emesse dall’ufficio comunale in Fast, perché deve essere un dipendente pubblico a certificare la situazione. Se mancano le squadre Fast, i geometri comunali possono accreditarsi in Regione ed i sopralluoghi speditivi con le relative ordinanze, possono trasformarsi in schede Fast. Poi queste devono diventare Aedes, che sono prodotte dalla perizia giurata dei tecnici. Chi ha la Fast con esito non utilizzabile, se ha un danno lieve deve poter presentare il progetto, fare i lavori e rientrare in pochi mesi. Le perizie giurate sono poi trasmesse all’ufficio per la ricostruzione e controllate”. Spuri ha precisato che per la ricostruzione leggera il termine dei lavori è di sei mesi e si possono eseguire opere di miglioramento sismico, purché non vengano allungati i tempi di lavoro. L’ingegnere ha annunciato che stanno per partire gli incarichi per attuare la microzonazione sismica. Infine ha detto che “Lunedì discuteremo dell’ordinanza relativa ai danni gravi, quando saranno emanate le norme, i professionisti potranno partire, questo non è condizionato da disponibilità economiche, perché il finanziamento è tramite il credito di imposta, con lo Stato che diviene debitore delle banche. Per le macerie si stanno attrezzando delle aree, in futuro la rimozione sarà a carico del privato con apposito rimborso”. Per chi ha avuto le schede prima del 30 ottobre, si può chiedere l’aggravamento del danno, presentando domanda in Comune, con perizia asseverata dal proprio tecnico, poi tutto sarà trasmesso all’ufficio ricostruzione per il controllo e la validazione.'
link: 'http://www.cronachemaceratesi.it/2017/02/18/spuri-le-casette-sono-da-evitare-chi-puo-scelga-lautonoma-sistemazione/926220/'
data: 20/02/2017
</yamldata></pre>

**Comment by chiccap** [comment]
@cristigalas io non inserirei

**Comment by cristigalas** [comment]
@chiccap No, non la inserirei. Questo sono notizie semmai da postare su Fb e Twitter ma non le pubblicherei sul sito anche perchè sono articoli lunghi. In questi casi bisognerebbe estrarre le nozitie di publbica utilità e quindi rimandare al link per la notizia completa. Ma è un lavoro un po' lungo e delicato che lascerei per le notizie davvero importanti. Del resto su questi tempi nei prossimi mesi saremo 'invasi' da notizie di tutti i tipi...
